packages/blade/src/components/Collapsible/_decisions/decisions.md

```diff
@@ -0,0 +1,186 @@
+  <CollapsibleBody>
+    <Box>Slot</Box>
+  </CollapsibleBody>
+</Collapsible>;
+
+<Collapsible>
+  <CollapsibleLink>Button</CollapsibleLink>
+  <CollapsibleBody>
+    <Box>Slot</Box>
+  </CollapsibleBody>
+</Collapsible>;
+```
+
+> **Note**
+>
+> - `Collapsible` will compose (`CollapsibleButton` or `CollapsibleLink`), `CollapsibleBody`. This ensures all interactions, a11y and animations work seamlessly at user end.
+> - `CollapsibleButton` and `CollapsibleLink` will support a subset of relevant props from `Button` and `Link` respectively.
+
+### Collapsible
+
+| Prop              | Type            | Default     | Description                                                        | Required |
+| ----------------- | --------------- | ----------- | ------------------------------------------------------------------ | -------- |
+| children          | `JSX`           | `undefined` | Composes `CollapsibleButton`, `CollapsibleLink`, `CollapsibleBody` | ✅       |
+| direction         | `bottom`, `top` | `bottom`    | Direction in which the content expands                             |          |
+| defaultIsExpanded | `boolean`       | `false`     | Expands the collapsible content by default (uncontrolled)          |          |
+| isExpanded        | `boolean`       | `undefined` | Expands the collapsible content (controlled)                       |          |
+| onChange          | `function`      | `undefined` | Callback for change in collapsible's expanded state                |          |
+
+> **Note**
+>
+> - Also includes layout based styling props
+> - For using collapsible in controlled state, use `isExpanded` coupled with `onChange`
+> - `onChange` callback signature `({ isExpanded }) => {}`, `isExpanded` represents the collapsible's current state
+
+### CollapsibleButton
+
+Built-in trigger for `Collapsible`, provides a subset of `Button` [properties](../../Button/_decisions/decisions.md).
+
+Supports ✅
+
+- `children`
+- `variant`
+- `size`
+- `icon`
+- `iconPosition`
+- `isDisabled`
+- `testID`
+- `accessibilityLabel`
+
+Doesn't support ❌
+
+- `isFullWidth` - can visually break UI
+- `onClick`, `isLoading` - using `onClick` will break built-in interactions (unless we provide additional hooks), `onChange` should already provide a way for users to listen-in on `Collapsible` state. This might have usage for async with `isLoading` in the future but for now not exposing the prop
+- `type` - will be `button` always
+
+### CollapsibleLink
+
+Built-in trigger for `Collapsible`, provides a subset of `Link` [properties](../../Link/_decisions/decisions.md).
+
+Supports ✅
+
+- `children`
+- `isDisabled`
+- `size`
+- `accessibilityLabel`
+- `testID`
+
+Doesn't support ❌
+
+- `icon` - renders a chevron icon always
+- `iconPosition` - will be `right` always for the chevron icon (which animates)
+- `htmlTitle`
+- `variant` - will always be `button`
+- `onClick`
+- `href`, `target`, `rel` - will always be a button
```

Comment:
We will still render a link here? or will it look like a link and work like a button?